main.go

```diff
@@ -103,6 +103,7 @@ func ApiServer(ctx context.Context) error {
 	signal.Notify(sig, os.Interrupt)
 
 	go func() {
+		log.Printf("listeing on port: %s", config.Port)
```

Comment:
```suggestion
		log.Printf("Listening on port: %s", config.Port)
```

Sebenernya kalo ini pake zerolog, lebih baik jadi:
```go
log.Info().Msgf("Listening on %s", net.JoinHostPort(config.Host, config.Port))
```